src/main/java/uk/gov/hmcts/darts/cases/controller/CaseController.java

```diff
@@ -63,16 +65,22 @@ public ResponseEntity<List<ScheduledCase>> casesGet(@NotNull @Parameter(name = "
-        @Size(max = 30) @Parameter(name = "defendant_name", description = "Full or partial Defendant name", in = ParameterIn.QUERY) @jakarta.validation.Valid @RequestParam(value = "defendant_name", required = false) String defendantName,
-        @Parameter(name = "date_from", description = "DateFrom to search for the hearings in.", in = ParameterIn.QUERY) @jakarta.validation.Valid @RequestParam(value = "date_from", required = false) @DateTimeFormat(iso = DateTimeFormat.ISO.DATE) LocalDate dateFrom,
-        @Parameter(name = "date_to", description = "DateTo to search for the hearings in.", in = ParameterIn.QUERY) @jakarta.validation.Valid @RequestParam(value = "date_to", required = false) @DateTimeFormat(iso = DateTimeFormat.ISO.DATE) LocalDate dateTo,
-        @Size(max = 100) @Parameter(name = "event_text_contains", description = "Full or partial event_text", in = ParameterIn.QUERY) @jakarta.validation.Valid @RequestParam(value = "event_text_contains", required = false) String eventTextContains
+        @Size(max = 20) @Parameter(name = "case_number", description = "Full or partial Case Number", in = ParameterIn.QUERY) @Valid @RequestParam(value = "case_number", required = false) String caseNumber,
+        @Size(max = 30) @Parameter(name = "courthouse", description = "Full or partial Courthouse name", in = ParameterIn.QUERY) @Valid @RequestParam(value = "courthouse", required = false) String courthouse,
+        @Size(max = 30) @Parameter(name = "courtroom", description = "Full or partial Courtroom name", in = ParameterIn.QUERY) @Valid @RequestParam(value = "courtroom", required = false) String courtroom,
+        @Size(max = 30) @Parameter(name = "judge_name", description = "Full or partial Judge name", in = ParameterIn.QUERY) @Valid @RequestParam(value = "judge_name", required = false) String judgeName,
+        @Size(max = 30) @Parameter(name = "defendant_name", description = "Full or partial Defendant name", in = ParameterIn.QUERY) @Valid @RequestParam(value = "defendant_name", required = false) String defendantName,
+        @Parameter(name = "date_from", description = "DateFrom to search for the hearings in.", in = ParameterIn.QUERY) @Valid @RequestParam(value = "date_from", required = false) @DateTimeFormat(iso = DateTimeFormat.ISO.DATE) LocalDate dateFrom,
+        @Parameter(name = "date_to", description = "DateTo to search for the hearings in.", in = ParameterIn.QUERY) @Valid @RequestParam(value = "date_to", required = false) @DateTimeFormat(iso = DateTimeFormat.ISO.DATE) LocalDate dateTo,
+        @Size(max = 100) @Parameter(name = "event_text_contains", description = "Full or partial event_text", in = ParameterIn.QUERY) @Valid @RequestParam(value = "event_text_contains", required = false) String eventTextContains
```

Comment:
Not sure these changes should be here?

Reply:
@cmagowan-hmcts would this cause any issues with your PR?

Reply:
These changes are already in master , so it should be fine.

Reply:
If that's the case they shouldn't be showing as a diff

Reply:
The diff is that  @jakarta.validation.Valid has been changed to @Valid , no point in having the qualifier when we are importing jakarta.validation.Valid;

Reply:
Ok, no problem.